### PR TITLE
Bug 1858010: decrease alerts severity

### PR DESCRIPTION
**Pull request by paulfantom**
* [ ] I added CHANGELOG entry for this change.
* [ ] No user facing changes, so no entry in CHANGELOG was needed.

/cc @openshift/openshift-team-monitoring @RiRa12621

### CODE REVIEW

**Review by RiRa12621 — APPROVED** [review]
/lgtm
Thank you

File: assets/prometheus-k8s/rules.yaml [hunk 1]
```diff
@@ -1325,17 +1325,41 @@ spec:
       for: 5m
       labels:
         severity: warning
+    - alert: KubeQuotaAlmostFull
```
Comment (lilic) [inline assets/prometheus-k8s/rules.yaml:1328]: This also introduces a new alert, not just decreases alert severity?

Comment (paulfantom) [reply assets/prometheus-k8s/rules.yaml:1328]: Yes, see https://coreos.slack.com/archives/C0VMT03S5/p1599729682146600

Comment (RiRa12621) [reply assets/prometheus-k8s/rules.yaml:1328]: By the name yes but effectively it's the same alert that existed just the with different names per threshold to account for the behavior that's happening.

**Comment by bmeng** [comment]
thank you for including that piece of change for backport :)

**Pull request merged**